src/YesSql.Abstractions/ConcurrencyException.cs

```diff
@@ -4,7 +4,8 @@ namespace YesSql
 {
     public class ConcurrencyException : Exception
     {
-        public ConcurrencyException() : base("The document could not be updated as it has been changed by another process.")
+        public ConcurrencyException(Document document) : 
```

Comment:
What about exposing the Document in a property of the exception? Maybe the caller could then extract anything they want from it, and could also log a precise error.

Reply:
That will be greate !, 

Because I did find some issues that were thrown out so far, it may have been rolled back, so I can't see the corresponding data content.